package/readme.txt

```diff
@@ -32,7 +32,7 @@ What's new since v0.9.7 ?
 -----------------------
 
 New features:
--
+- Support opening compressed files in the XZ file format.
```

Comment:
this can now indicate that compression is also supported with something like "Support compression and decompression of XZ archives"

Reply:
Updated